pkg/plugins/policies/core/matchers/testdata/matchedpolicies/meshgateways/01.golden.yaml

```diff
@@ -39,67 +39,133 @@ FromRules:
-  127.0.0.1:8081:
-  - Conf:
-      backends:
-      - file:
-          path: /servicesubset
-        type: File
-    Origin:
-    - creationTime: "0001-01-01T00:00:00Z"
-      mesh: mesh-1
-      modificationTime: "0001-01-01T00:00:00Z"
-      name: gateway
-      type: MeshAccessLog
-    - creationTime: "0001-01-01T00:00:00Z"
-      mesh: mesh-1
-      modificationTime: "0001-01-01T00:00:00Z"
-      name: mesh
-      type: MeshAccessLog
-    - creationTime: "0001-01-01T00:00:00Z"
-      mesh: mesh-1
-      modificationTime: "0001-01-01T00:00:00Z"
-      name: servicesubset
-      type: MeshAccessLog
-    Subset: []
-  127.0.0.1:8082:
-  - Conf:
-      backends:
-      - file:
-          path: /to-gateway
-        type: File
-    Origin:
-    - creationTime: "0001-01-01T00:00:00Z"
-      mesh: mesh-1
-      modificationTime: "0001-01-01T00:00:00Z"
-      name: gateway
-      type: MeshAccessLog
-    - creationTime: "0001-01-01T00:00:00Z"
-      mesh: mesh-1
-      modificationTime: "0001-01-01T00:00:00Z"
-      name: mesh
-      type: MeshAccessLog
-    Subset: []
+  ByListener:
+    127.0.0.1:8080:
+    - Conf:
+        backends:
+        - file:
+            path: /gateway-listener
+          type: File
+      Origin:
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: gateway
+        type: MeshAccessLog
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: gatewaylistener
+        type: MeshAccessLog
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: mesh
+        type: MeshAccessLog
+      Subset: []
+    127.0.0.1:8081:
+    - Conf:
+        backends:
+        - file:
+            path: /servicesubset
+          type: File
+      Origin:
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: gateway
+        type: MeshAccessLog
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: mesh
+        type: MeshAccessLog
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: servicesubset
+        type: MeshAccessLog
+      Subset: []
+    127.0.0.1:8082:
+    - Conf:
+        backends:
+        - file:
+            path: /to-gateway
+          type: File
+      Origin:
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: gateway
+        type: MeshAccessLog
+      - creationTime: "0001-01-01T00:00:00Z"
+        mesh: mesh-1
+        modificationTime: "0001-01-01T00:00:00Z"
+        name: mesh
+        type: MeshAccessLog
+      Subset: []
+  ByListenerAndHostname:
+    127.0.0.1:8080:*:
```

Comment:
Isn't this notation problematic for ipv6? should we use a different delimiter?

Reply:
We only serialize this for tests